e2e/docker-compose.yml

```diff
@@ -15,6 +15,7 @@ services:
       - "-rpcallowip=0.0.0.0/0"
       - "-rpcbind=0.0.0.0:18443"
       - "-txindex=1"
+      - "-rpcworkqueue=400"
```

Comment:
this is likely not needed, but it doesn't hurt; I was seeing a few warnings of it being too small to handle requests so I bumped it up